app/views/content/life-as-a-teacher/teaching-as-a-career/from-qualified-teacher-to-head-of-biology/_article.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+      <h2>I knew teaching was for me</h2>
+      <p>From an early age, I knew a teaching career was for me. I could not imagine working a typical nine-to-five desk job. Every day in teaching brings a new challenge and new situations, which is what makes the job so exciting.</p>
```

Comment:
should this be `9-to-5`? or `9am to 5pm`?

but happy for you to leave as is if that's best practice!

Reply:
We've changed it to 9 to 5 - let me know if you're happy with that?